integrated_channels/canvas/admin/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""
+Canvas admin integration for configuring Canvas app to communicate with Canvas systems.
+"""
+
+from django.contrib import admin
+
+from integrated_channels.canvas.models import CanvasEnterpriseCustomerConfiguration
+
+
+@admin.register(CanvasEnterpriseCustomerConfiguration)
+class CanvasEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
+    """
+    Django admin model for CanvasEnterpriseCustomerConfiguration.
+    """
+    list_display = (
+        "enterprise_customer_name",
+        "client_id",
+        "client_secret",
+        "canvas_account_id",
+        "canvas_base_url",
```

Comment:
so this is globalconfig not enterprise customer, but how/where is the `course_api_path` field going to be set?

Reply:
`canvas_base_url` is enterprise customer specific and stored in the canvas enterprise customer config. I can add another admin form for the `course_api_path` but I have more overarching questions as to why we're storing that field in the model instead of something like the app config. Does this path change? Is it user specific?

Reply:
I figure we can do without an admin for for that for now since we assume it will never change, but (everything changes in real life), so I would be down with the idea of handling this in the future (either by moving it to app config, or creating a django form)

Reply:
same applies to oauth_api_path etc that are not enterprise specific

Reply:
Decided that the global config that contains the course api path was entirely unnecessary so I've removed it.